### ISSUE TITLE
Completing the code for the model.

### ISSUE THREAD

**Issue by kartikdutt18** [rcurtin/vulgarnet#1]
Hey @rcurtin, This would be a really cool implementation especially if we use mlpack. Do you mind if I work on it in my spare time, when I get it working I could create a PR. Let me know what you think. Also if you had any specifics in mind that would be great. Thanks a lot.

**Comment by rcurtin** [comment]
Ha!  Neat that you found this.  I had forgotten about it.  I haven't touched it in 5 years... it probably would be better to start over entirely.  I think some of the new string loading support could be really helpful for a task like this.

**Comment by kartikdutt18** [comment]
> I think some of the new string loading support could be really helpful for a task like this.

Yeah, I'll spend some time with them, then maybe we can create a simple model and make a submission [here](https://www.kaggle.com/c/jigsaw-multilingual-toxic-comment-classification/data) or at least showcase a real world example with mlpack.
